examples/plot_clip.py

```diff
@@ -0,0 +1,156 @@
+"""
+Clip Vector Data with GeoPandas
+==================================================================
+
+Learn how to clip point, line, or polygon geometries to the boundary
+of a polygon geometry using GeoPandas.
+
+"""
+
+###############################################################################
+# Clip Vector Data in Python Using GeoPandas
+# ---------------------------------------------
+#
+# .. note::
+#    The example below will show you how to use the clip() function to clip
+#    vector data such as points, lines and polygons to a vector boundary.
+#
+# The example below walks you through a typical workflow for clipping one
+# vector data file to the shape of another. Both vector data files must be
+# opened with GeoPandas as GeoDataFrames and be in the same Coordinate
+# Reference System (CRS) for the ``clip_shp()`` function in EarthPy to work.
+#
+# This example uses Polygons, a Line, and Points made with shapely and then
+# turned into GeoDataframes.
+#
+# .. note::
+#    The object to be clipped will be clipped to the full extent of the clip
+#    object. If there are multiple polygons in clip object, the input data will
+#    be clipped to the total boundary of all polygons in clip object.
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages. You will primarily use EarthPy's clip
+# utility alongside GeoPandas.
+
+import numpy as np
+import matplotlib.pyplot as plt
+import geopandas as gpd
+from shapely.geometry import Polygon, LineString, Point
+import geopandas.clip as gc
+
+###############################################################################
+# Create Example Data
+# -------------------
+#
+# Once the packages have been imported, you need to create the shapes to clip.
+# You need to make two polygons, one line, and one point feature with shapely,
+# and then open those shapes up with GeoPandas.
```

Comment:
you can delete this. just say you're creating data for the vignette and remove lines 49-50. i suggested some language above!